### ISSUE TITLE
ArgumentNullException when parsing PL/SQL file

### ISSUE THREAD

**Issue by t-rapp** [NaturalDocs/NaturalDocs#59]
Crash Message:

   Der Wert darf nicht NULL sein.
Parametername: value (Parser Thread 3)
   (System.ArgumentNullException)

Task:

   Parsing file D:\Temp\NaturalDocs-Test01\Packages\FOO_ENGINE Body.sql

Stack Trace:
```
   ----- Parser Thread 3 -----
   bei System.String.IndexOf(String value, Int32 startIndex, Int32 count, StringComparison comparisonType)
   bei CodeClear.NaturalDocs.Engine.Languages.Parsers.SQL.TryToFindBasicPrototype(Topic topic, LineIterator startCode, LineIterator endCode, TokenIterator& prototypeStart, TokenIterator& prototypeEnd)
   bei CodeClear.NaturalDocs.Engine.Languages.Language.AddBasicPrototypes(Tokenizer source, List`1 elements, List`1 possibleDocumentationComments)
   bei CodeClear.NaturalDocs.Engine.Languages.Language.Parse(Tokenizer source, Int32 fileID, CancelDelegate cancelDelegate, IList`1& topics, LinkSet& classParentLinks)
   bei CodeClear.NaturalDocs.Engine.Languages.Language.Parse(Path filePath, Int32 fileID, CancelDelegate cancelDelegate, IList`1& topics, LinkSet& classParentLinks)
   bei CodeClear.NaturalDocs.Engine.Files.ChangeProcessor.ProcessNewOrChangedSourceFile(File file, Accessor codeDBAccessor, CancelDelegate cancelDelegate)
   bei CodeClear.NaturalDocs.Engine.Files.ChangeProcessor.ProcessNewOrChangedFile(File file, Accessor codeDBAccessor, CancelDelegate cancelDelegate)
   bei CodeClear.NaturalDocs.Engine.Files.ChangeProcessor.WorkOnProcessingChanges(CancelDelegate cancelDelegate)
   bei CodeClear.NaturalDocs.Engine.Thread.InternalStart()
   ----- Parent Thread -----
   bei CodeClear.NaturalDocs.CLI.Application.Multithread(String threadName, CancellableTask task)
   bei CodeClear.NaturalDocs.CLI.Application.BuildDocumentation(ErrorList errorList)
   bei CodeClear.NaturalDocs.CLI.Application.Main(String[] commandLine)
```
Command Line:

   "C:\Program Files (x86)\Natural Docs\NaturalDocs.exe" --input ../Packages --output HTML ./Html --project ./Project

Versions:

   Natural Docs 2.1
   Windows 7 Professional
   .NET 4.5.2
   SQLite 3.31.1

Input File (reduced):

```sql
CREATE OR REPLACE PACKAGE BODY FOO_ENGINE AS

---tasks
FUNCTION GetDefinedTaskStateList(UserID INTEGER) RETURN XMLTYPE IS
  v_xml XMLTYPE;
BEGIN
  RETURN v_xml;
END;

END FOO_ENGINE;
```

**Comment by t-rapp** [comment]
The exception disappears when I change the comment line into:
```
-- tasks
```

**Comment by NaturalDocs** [comment]
See if this version fixes it: https://www.naturaldocs.org/download/temp/Natural_Docs_temp1.zip

**Comment by t-rapp** [comment]
Thanks, the project is generated fine now with that fix.

**Comment by NaturalDocs** [comment]
This fix will be included in the next releases: Natural Docs 2.2 (Development Release 2) and Natural Docs 2.2 proper.  The link to the zip file will work until then.
